src/lxc/lxccontainer.c

```diff
@@ -1320,7 +1322,7 @@ static bool create_run_template(struct lxc_container *c, char *tpath, bool need_
 		}
 		/* execute */
 		execvp(tpath, newargv);
-		SYSERROR("failed to execute template %s", tpath);
+		SYSERROR("Failed to execute template %s", tpath);
 		exit(1);
```

Comment:
@brauner The first letter of the message should be uppercase or lowercase? I noticed that sometimes you change uppercase to lowercase and sometimes you change lowercase to uppercase.